tests/general/sigabrt.c

```diff
@@ -0,0 +1,11 @@
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(void)
+{
```

Comment:
The style here seems to be curlys on the same line as the declaration?

Reply:
Fixed.